eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/ActiveClientTokenManager.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs.implementation;
+
+import com.azure.core.amqp.CBSNode;
+import com.azure.core.amqp.exception.AmqpResponseCode;
+import com.azure.core.implementation.logging.ServiceLogger;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.FluxSink;
+import reactor.core.publisher.Mono;
+
+import java.io.Closeable;
+import java.time.Duration;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Manages the re-authorisation of the client to the token audience against the CBS node.
```

Comment:
I will only accept `authorisation` as a valid spelling if you agree to also use `colour` and `flavour`

Reply:
Of course. ;) I'm Canadian after all! Are there any other ways to spell it? 🤣

Reply:
Cries. I wish we could teach the US to spell properly, but I do tend to argue for the incorrect US spelling in our reference docs, just for consistency.